### ISSUE TITLE
Deprecation Warning: Instead of sudo/sudo_user, use become/become_user

### ISSUE THREAD

**Issue by dianaarnos** [phansible/phansible#273]
I know you guys will remove all warnings, but I guess it's a good idea to help you list some:
```
==> default: [DEPRECATION WARNING]: Instead of sudo/sudo_user, use become/become_user and ==> default: make sure become_method is 'sudo' (default). This feature will be removed in a ==> default: future release. Deprecation warnings can be disabled by setting ==> default: deprecation_warnings=False in ansible.cfg.
```

Ansible version: 2.0.1.0

**Comment by debo** [comment]
Hi @dianaarnos thanks a lot for the report, appreciated.

**Comment by naxhh** [comment]
I think we are overusing sudo too, so will be nice to check for that too. But maybe more in the future with more time and better CI.

**Comment by debo** [comment]
Indeed @dianaarnos, feel free to contribute, especially in the meanwhile I have no connectivity in my new place; I hope it gets sorted soon.

**Comment by dianaarnos** [comment]
For sure, I will, @naxhh and @debo. I will take a look at it this week (after I finish some workload that is killing me right now)

**Comment by dianaarnos** [comment]
Started working on this one today :)

**Comment by debo** [comment]
@dianaarnos that's great news and to add joy to the great news my internet connection is getting sorted so I will be back soon too and able to carry on the work I started.

**Comment by dianaarnos** [comment]
I'm facing a strange problem here...
When I run the project and generate my phansible zip, asking just for installing php, my playbook.yml has the following roles:
```
  roles:
    - server
    - vagrant_local
    - php
    - app
```
But the only role generated is the `app` one. 
I'm running from branch `master`.
Did anyone had the same problem?

**Comment by debo** [comment]
Hi @dianaarnos what do you mean exactly? Are the other three roles empty? Production is likely running on the latest tag [0.7.2](https://github.com/phansible/phansible/releases/tag/0.7.2)

**Comment by dianaarnos** [comment]
Yeah, the roles are empty. Actually, the folders for the roles are not even created, only the `app` one.
Later today I'll checkout the tag and try again

**Comment by debo** [comment]
Production works fine for sure, I'll check master in a moment. I'm in a cafe nearby home so I better get the most out of their internet connection :P

**Comment by debo** [comment]
@dianaarnos I found the root of your problem. Because of PR https://github.com/phansible/phansible/pull/254 all the roles have been now moved to their own repos. We would like people to contribute to the roles individually and we then take care of restoring the old behaviour cloning those roles in the right place during deployment of the website. (more reasoning in the PR comments)

For the time being if you can't avoid to use the phansible repo use [this gist])https://gist.github.com/272693bc599b1a69d2c9) I quickly put together for you. Just run it from the project root with `bash local-deploy` and then launch the site locally and everything should then work just fine.

I hope it helps.

**Comment by dianaarnos** [comment]
Ah, that solves my problem, thanks!
Now everything makes sense, lol.
I just need it to generate the files and try to do a `vagrant up` just to guarantee I didn't forget any piece of `sudo` somewhere.

**Comment by dianaarnos** [comment]
I need to ask a question just by curiosity: why not use git submodules for that?

**Comment by naxhh** [comment]
Ups!! That was my fault :(

I thought I've added https://github.com/phansible/phansible/blob/master/ansible/roles/app/tasks/main.yml to the Vagrant execution.. seems I forget.

We tried git submodules in the past. They were a bit of a pain to configure and work with. We didn't like them at all see https://github.com/phansible/phansible/issues/41

We moved again to other repos because makes CI and sharing responsabilities much easier. We hope this approach is better than with submodules. But we will see.

**Comment by debo** [comment]
@dianaarnos I personally never used git submodules because I never had the need to but we could have a look at it, however if we put the app task in place as @naxhh mentioned that should do the trick without the need. I'm open to suggestion though.

**Comment by dianaarnos** [comment]
@naxhh it's not there, but I was running phansible localy, anyway. That local-deploy script worked just fine.
And I can understand the reason behind the separate modules instead of submodules. It also makes things easier for everyone.

**Comment by dianaarnos** [comment]
I'll make a PR for each role linking to this issue and then I'll make a PR for this repo, linking all the other PRs on description.

**Comment by debo** [comment]
@dianaarnos one of the reason we also wanted to keep those roles separates it's also to give a curated, tested, and trusted set of resources that advanced phansible users (as in php ansible users) can rely on without the need to pass via the wensite frontend. I personally often struggled in finding the right role, when you on galaxy or other resources you always find 50 hits of the same role and you never know which one to pick, we would like to also solve that problem without losing the simplicity that phansible.com provides as bootstrap.

By the way thanks a lot for your PRs and contribution

**Comment by naxhh** [comment]
I've merged everything with a green CI. I let the rest of the PR's to @debo's considerations.
Thanks for taking the time @dianaarnos

**Comment by dianaarnos** [comment]
You're welcome :)
Thank you for helping maintaining this awesome project.

**Comment by naxhh** [comment]
Ups. closed by mistake

**Comment by debo** [comment]
@dianaarnos are you happy for me to close this one?
